xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-wiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -168,7 +168,7 @@ platform.wiki.currentwiki.candidates.hint=Users who have requested or been invit
 
 # ADMIN
 admin.xwiki.manageworkspace.description=Manage settings such as wiki name, description, owner and membership type.
-admin.xwiki.workspaceusers.description=Manage wiki members: add, remove, invite, accept or reject wiki members.
+admin.xwiki.workspaceusers.description=Manage wiki members: add, remove, invite, wiki members & accept or reject join requests.
```

Comment:
I don't see what is wrong with the previous version. Both `accept` and `reject` refer to wiki members. The new version doe not even make sense to me.

Reply:
Yeah, looking over it again and doesn't sound right. Have reverted it.